perfrepo-cli

```diff
@@ -157,18 +157,18 @@ class TestCLI(GenericCLI):
             try:
                 test_id = int(argv[1])
                 test = self._perf_api.test_get_by_id(test_id)
-                if test is None:
-                    test = self._perf_api.test_get_by_uid(test_id)
-                if test is None:
-                    print("Test not found.")
-                else:
-                    print(test)
             except IndexError:
                 print("No ID specified!", file=sys.stderr)
                 return -1
             except ValueError:
-                print("ID needs to be an integer!", file=sys.stderr)
-                return -1
+                test = self._perf_api.test_get_by_uid(argv[1])
+
+            if test is None:
+                print("Test not found.")
+                return 0
```

Comment:
I also disagree with this... we've discussed this in person, it should be a negative value so that scripts can easily detect by the return value if the test was or wasn't found... but the help message shouldn't be printed in such a case.